### PR TITLE
Feature/individual ad

### PR DESCRIPTION
**Pull request by juancoelhoo**
I implemented the advertisement pop-up and the dropdown menu.

### CODE REVIEW

File: frontend/src/components/PedAd-Popup/PedAd-Popup.css
Comment (beviggiano) [inline frontend/src/components/PedAd-Popup/PedAd-Popup.css:1]: No comments on CSS files please

File: frontend/src/pages/Advertisement/Advertisement.css
Comment (beviggiano) [inline frontend/src/pages/Advertisement/Advertisement.css:1]: Here as well

**Review by beviggiano — CHANGES_REQUESTED** [review]
Just some points about CSS files

**Review by beviggiano — COMMENTED** [review]
It's all ok for me

**Review by beviggiano — APPROVED** [review]

File: frontend/src/components/PetAd/PetAd.tsx [hunk 1]
```diff
@@ -1,26 +1,35 @@
...
       <img src={dogimage} alt="pet-photo" />
       <div className="pet-inf">
         <div className="pet-name">
           <img src={nameimage} alt="" />
-          Nome: <span className="pet-info">caramelo</span>
+          Nome: <span className="pet-info">{name}</span>
         </div>
         <div className="pet-breed">
           <img src={breedimage} alt="" />
           Raça: <span className="pet-info">Pastor alemão</span>
```
Comment (edu-correia) [inline frontend/src/components/PetAd/PetAd.tsx:28]: change "Pastor alemão" to {breed}

File: frontend/src/components/PetAd/PetAd.tsx [hunk 2]
```diff
@@ -1,26 +1,35 @@
...
         </div>
         <div className="pet-age">
           <img src={ageimage} alt="" />
-          Idade: <span className="pet-info">2 anos</span>
+          Idade: <span className="pet-info">{age} anos</span>
         </div>
         <div className="pet-description">
           <img src={descimage} alt="" />
```
Comment (edu-correia) [inline frontend/src/components/PetAd/PetAd.tsx:35]: change "Lorem, ipsum dolor sit amet" to {description} in the line below

**Review by edu-correia — CHANGES_REQUESTED** [review]
Just 2 points to change

File: frontend/src/components/PetAd/PetAd.tsx (hunk 1 above)
Comment (juancoelhoo) [reply frontend/src/components/PetAd/PetAd.tsx]: Thanks!

**Review by beviggiano — APPROVED** [review]

**Review by edu-correia — APPROVED** [review]
LGTM

**Pull request merged**